lint-js.sh

```diff
@@ -21,25 +21,7 @@ fi
 
 #Go to wp-content directory.
 cd "$wpcontentdir";
-
-# Make sure that packagejson and composer json exist in wp-content.
-if [ ! -f package.json ] || [ ! -f composer.json ]; then
-	cd "$scriptsdir";
-	sh hoister.sh -c "$wpcontentdir";
-	cd "$wpcontentdir";
-fi
-
-# Make sure that node_modules exists in wp-content.
-if [ ! -d node_modules ] || [ ! -d vendor ]; then
-	# Run npm install in wp-content
-	npm install;
-fi
-
-# Make sure that vendor exists in wp-content.
-if [ ! -d vendor ]; then
-	# Run composer install in wp-content
-	composer install;
-fi
```

Comment:
This is also in `lint-css.sh`, which was fine when there were only 2. 

But I felt strange about copy-pasting it into `test-js.sh` also. 

This could be needlessly DRY, though 🤣